odoo/tools/pdf/signature.py

```diff
@@ -78,7 +78,8 @@ def _load_key_and_certificate(self) -> tuple[Optional[PrivateKeyTypes], Optional
         Returns:
             Optional[PrivateKeyTypes]: a private key object, or None if the key couldn't be loaded.
         """
-        if "signing_certificate_id" not in self.company or not self.company.signing_certificate_id:
+        if ("signing_certificate_id" not in self.company or not self.company.signing_certificate_id
+            or not self.company.signing_certificate_id.pem_certificate):
```

Comment:
```suggestion
        if "signing_certificate_id" not in self.company \ 
            or not self.company.signing_certificate_id \
            or not self.company.signing_certificate_id.pem_certificate:
```

Make easy to readable